src/functional/ffront/ast_passes/remove_docstrings.py

```diff
@@ -0,0 +1,71 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import ast
+import dataclasses
+
+
+@dataclasses.dataclass(kw_only=True)
+class RemoveDocstrings(ast.NodeTransformer):
+    """
+    Remove ast docstring from body of ast.FunctionDef.
+
+    Docstrings appear as type ast.Expr with ast.Constant value of type string.
+    If such patterns is detected, this entry in the node.body list is removed.
```

Comment:
```suggestion
```
I think this is too detailed. After all you just describe the `if` condition below. After reading this the condition is not easier to understand and there is no additional information, so I would just remove it (avoiding the two to become desync).